app/assets/stylesheets/govuk_publishing_components/components/_attachment.scss

```diff
@@ -5,7 +5,7 @@ $thumbnail-background: govuk-colour("white");
 $thumbnail-border-colour: rgba(11, 12, 12, .1);
 $thumbnail-shadow-colour: rgba(11, 12, 12, .4);
 $thumbnail-shadow-width: 0 2px 2px;
-$thumbnail-icon-border-colour: govuk-colour("grey-3");
+$thumbnail-icon-border-colour: govuk-colour("mid-grey");
```

Comment:
If you want these components to work in downstream applications with compatibility mode enabled, you'll want to pass `$legacy: "grey-3"` here so that it knows which colour to use from the legacy palette, whenever you use a colour that's been renamed or added.

Without this, if you tried to use the attachment component in an app that has compatibility mode enabled, you'd get an 'Unknown colour mid-grey' error.

http://govuk-frontend-review.herokuapp.com/docs/#function-govuk-colour

You might also want to find a way to surface or test with legacy mode within publishing components, so that you don't find out things like this are broken after you've published!

Reply:
Sounds good. Wasn't aware of the `$legacy` attribute - haven't seen it anywhere yet.

Reply:
Adressed in 4c479d7